### PR TITLE
Nlb vnf route mode

### PR DESCRIPTION
**Pull request by ujjwal-ibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMISLB_basic_network_vnf
--- PASS: TestAccIBMISLB_basic_network_vnf (989.38s)
PASS
ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm	992.835s
```

### CODE REVIEW

File: website/docs/r/is_lb.html.markdown [hunk 1]
```diff
@@ -47,6 +47,8 @@ Review the argument references that you can specify for your resource.
 - `name` - (Required, String) The name of the VPC load balancer.
 - `profile` - (Required, Forces new resource, String) The profile to use for this load balancer. Supported value is `network-fixed`.
 - `resource_group` - (Optional, Forces new resource, String) The resource group where the load balancer to be created.
+- `route_mode` - (Optional, Forces new resource, Bool) Indicates whether route mode is enabled for this load balancer.
+  **NOTE** Currently, public load balancers are not supported with `route_mode` enabled.
```
Comment (SunithaGudisagarIBM) [inline website/docs/r/is_lb.html.markdown:51]: DO we need to specify 'profile' must be 'network-fixed' also here?

File: ibm/resource_ibm_is_lb.go [hunk 2]
```diff
@@ -156,6 +163,13 @@ func resourceIBMISLB() *schema.Resource {
 				Computed: true,
 			},
 
+			isLBRouteMode: {
```
Comment (SunithaGudisagarIBM) [inline ibm/resource_ibm_is_lb.go:166]: description for isLBRouteMode is missing

File: website/docs/r/is_lb.html.markdown (hunk 1 above)
Comment (deepaksibm) [reply website/docs/r/is_lb.html.markdown:51]: refer this change for 'profile' doc change. https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3170/files. 'profile' doc update will go as part of this pr

**Comment by astha-jain** [comment]
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3207 Merged

**Pull request closed without merging**